apiserver/app/apiKey.go

```diff
@@ -12,6 +14,22 @@ import (
 	"github.com/slovak-egov/einvoice/pkg/handlerutil"
 )
 
+func getIntClaim(claims jwt.MapClaims, key string) (int, error) {
+	rawValue, ok := claims[key]
+	if !ok {
+		return 0, errors.New(fmt.Sprintf("Key '%v' not found in claims", key))
+	}
+	switch v := rawValue.(type) {
+	case float64:
+		if math.Mod(v, 1.0) == 0 {
+			return int(v), nil
+		}
+	case int:
+		return v, nil
+	}
+	return 0, errors.New(fmt.Sprintf("Key '%v' in claims has wrong type", key))
+}
+
 func (a *App) getUserIdByApiKey(ctx goContext.Context, tokenString string) (int, error) {
 	var user *entity.User
 	token, err := jwt.Parse(tokenString, func(token *jwt.Token) (interface{}, error) {
```

Comment:
hmm, what about using `jwt.Parser{UseJSONNumber: true}`
then we can just cast to`json.Number` and apply function `.Int64()` (https://golang.org/pkg/encoding/json/#Number.Int64)
and we will avoid casing `float64` and weird modulus by `1.0`